### ISSUE TITLE
problem when classes are not presented in the val.set 

### ISSUE THREAD

**Issue by Houssembenmid** [intel-isl/Open3D-PointNet2-Semantic3D#75]
I have trained my own data that is similar to semantic3d with 24 class instead of 9 . When training it appears that some of the IOU perclass having values above 0.5 the problem is that this does not make sense because they simply do not exist in the validation set only in the training set. I want to know how the IOU per class is calculated in the metri.py file because as I know it suppose to be NaN value or 1 with a smoothing factor .
please help since I am new to coding and I am really struggling understanding how the IOU per class was calculated .

Thank you very much.
